Documentation/intro-to-clr.md

```diff
@@ -202,7 +202,7 @@ As an aside, while exceptions avoid one common error (not checking for failure),
 
 ### Parameterized Types (Generics)
 
-Previous to version 2.0 of the CLR, the only parameterized types were arrays.  All other containers (such as hash tables, lists, queues, etc.), all operated on a generic Object type.  The inability to create List<ElemT>, or Dictionary<KeyT, ValueT> certainly had a negative performance effect because value types needed to be boxed on entry to a collection, and explicit casting was needed on element fetch.  Nevertheless, that is not the overriding reason for adding parameterized types to the CLR.  The main reason is that **parameterized types make programming easier**.
+Previous to version 2.0 of the CLR, the only parameterized types were arrays.  All other containers (such as hash tables, lists, queues, etc.), all operated on a generic Object type.  The inability to create List<ElemT>, or Dictionary<KeyT, ValueT> certainly had a negative performance effect because value types needed to be boxed on entry to a collection, and explicit casting was needed on element fetch.  Nevertheless, that is not the overriding reason for adding parameterized types to the CLR.  The main reason is that **parameterized types make programming easier** by enabling static type-checking at compilation phase and enforcing type-safety.
 
```

Comment:
I meant CIL to native compilation but CSC-to-CIL holds too. I will drop the change and try to add a new paragraph after I become more familiar with the project :)